packages/seo-store/src/analysis/slice/results.js

```diff
@@ -52,39 +50,65 @@ const initialState = {
 	status: ASYNC_STATUS.IDLE,
 	error: "",
 	seo: {
-		focus: {},
+		focus: {
+			score: 0,
+			results: [],
+		},
+	},
+	readability: {
+		score: 0,
+		results: [],
 	},
-	readability: {},
-	research: {
-		morphology: {},
+	research: {},
+	activeMarker: {
+		id: "",
+		marks: [],
```

Comment:
Why was this needed again? 🤔 

In theory the marker IDs should be unique and we could loop through the SEO and readability results to find the corresponding marks there.

What do you think?

Reply:
I think ideally yes, we would only keep track of `activeMarkerId`. But let's see how easy this is to implement once integrating.